projects/tinkerbell/hegel/Makefile

```diff
@@ -0,0 +1,35 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.15"
+
+REPO=hegel
+REPO_OWNER=tinkerbell
+
+LICENSE_PACKAGE_FILTER?=./cmd/hegel
```

Comment:
I think you can drop this since it matches SOURCE_PATTERNS, it gets set to that by default